api/CharacterData.json

```diff
@@ -65,7 +65,7 @@
             },
             "firefox": {
               "version_added": "25",
-              "notes": "Two properties, <code>nextElementSibling</code> and <code>previousElementSibling</code>, have been moved to the <a href='https://developer.mozilla.org/docs/Web/API/NonDocumentTypeChildNode><code>NonDocumentTypeChildNode</code></a> interface, also implemented by <code>CharacterData</code>."
+              "notes": "Two properties, <code>nextElementSibling</code> and <code>previousElementSibling</code>, have been moved to the <a href='https://developer.mozilla.org/docs/Web/API/NonDocumentTypeChildNode'><code>NonDocumentTypeChildNode</code></a> interface, also implemented by <code>CharacterData</code>."
```

Comment:
This `href` attribute had an unclosed `'`, which broke the compatibility table: https://developer.mozilla.org/docs/Web/API/CharacterData#Browser_compatibility